src/jsx.d.ts

```diff
@@ -2,12 +2,117 @@
 /// <reference lib="dom" />
 
 import {
-	ClassAttributes,
 	Component,
 	ComponentType,
 	FunctionComponent,
-	PreactDOMAttributes,
-	VNode
+	VNode,
+	SVGAttributes,
+	HTMLAttributes,
+	AnnotationMathMLAttributes,
+	AnnotationXmlMathMLAttributes,
```

Comment:
This mega import statement is kinda gross but I'm not sure an `import * as domish` is all that much better?